templates/ansible/hieradata.j2

```diff
@@ -53,14 +50,11 @@ aem_curator::install_author::data_volume_device: "{{ aws[os_type].devicemap_data
 
 aem_curator::install_publish::aem_profile: {{ aem.profile }}
 aem_curator::install_publish::aem_keystore_password: "%{lookup('{{ aem.keystore_password_parameter }}')}"
+aem_curator::install_publish::aem_ssl_method: {{ aem.ssl_method }}
 aem_curator::install_publish::aem_artifacts_base: {{ aem.artifacts_base }}
 aem_curator::install_publish::aem_jvm_mem_opts: "{{ aem.publish.jvm_mem_opts }}"
 aem_curator::install_publish::aem_jvm_opts:
 - '-XX:+PrintGCDetails'
-- '-XX:+PrintGCTimeStamps'
-- '-XX:+PrintGCDateStamps'
-- '-XX:+PrintTenuringDistribution'
-- '-XX:+PrintGCApplicationStoppedTime'
```

Comment:
same as above

Reply:
It is only needed for JDK11, I will make this change.